pkgs/applications/video/lbry/default.nix

```diff
@@ -0,0 +1,65 @@
+{ lib
+, fetchurl
+, stdenv
+, appimageTools
+, ffmpeg
+, gtk3
+, makeWrapper
+, xdg_utils
```

Comment:
```suggestion
, xdg-utils
```

Reply:
Ah, xdg_utils is now an alias. Makes sense.

Reply:
BTW, while importing these changes I found another instance of xdg_utils, so self-importing makes sense here.